src/wp-includes/class-wp-duotone.php

```diff
@@ -948,45 +944,45 @@ public static function register_duotone_support( $block_type ) {
 	 * This handles selectors defined in `color.__experimentalDuotone` support
 	 * if `filter.duotone` support is not defined.
 	 *
-	 * @param string $block_name The block name.
+	 * @param WP_Block_Type $block_type Block type to check for support.
 	 *
 	 * @internal
 	 *
 	 * @since 6.3.0
 	 *
 	 * @return string The CSS selector or null if there is no support.
```

Comment:
```suggestion
	 * @internal
	 *
	 * @since 6.3.0
	 *
	 * @param WP_Block_Type $block_type Block type to check for support.
	 * @return string The CSS selector or null if there is no support.
```